tls/s2n_server_new_session_ticket.c

```diff
@@ -280,7 +328,13 @@ S2N_RESULT s2n_tls13_server_nst_write(struct s2n_connection *conn, struct s2n_st
     RESULT_GUARD_POSIX(s2n_stuffer_reserve_uint24(output, &message_size));
 
     uint32_t ticket_lifetime_in_secs = 0;
-    RESULT_GUARD(s2n_generate_ticket_lifetime(conn, &ticket_lifetime_in_secs));
+    RESULT_GUARD(s2n_generate_ticket_lifetime(conn, key->intro_timestamp, key_retrieval_time, &ticket_lifetime_in_secs));
+
+    /**
+     *= https://www.rfc-editor.org/rfc/rfc8446#section-4.6.1
+     *# The value of zero indicates that the ticket should be discarded immediately.
+     **/
```

Comment:
This part of the RFC is related to lifetimes read from tickets. Does it apply here?

If you're trying to say that if we send this ticket then it will immediately be discarded by the peer, then that deserves a comment but probably not a compliance comment. This isn't where we implement the logic to discard expired tickets.